deepfence_server/model/scans.go

```diff
@@ -22,22 +22,25 @@ type MalwareScanTriggerReq struct {
 }
 
 type ComplianceScanTriggerReq struct {
-	ScanTriggers []ComplianceScanTrigger `json:"scan_triggers" required:"true"`
+	ScanTriggerCommon
+	ComplianceBenchmarkTypes
 }
 
-type KeyValue struct {
-	Key   string `json:"key" required:"true"`
-	Value string `json:"value" required:"true"`
+type KeyValues struct {
+	Key    string   `json:"key" required:"true"`
+	Values []string `json:"values" required:"true"`
 }
 
 type FieldsFilter struct {
-	FieldsValues []KeyValue `json:"fields_values" required:"true"`
+	FieldsValues []KeyValues `json:"fields_values" required:"true"`
 }
 
 type ScanFilter struct {
-	ImageScanFilter     FieldsFilter `json:"image_scan_filter" required:"true"`
-	ContainerScanFilter FieldsFilter `json:"container_scan_filter" required:"true"`
-	HostScanFilter      FieldsFilter `json:"host_scan_filter" required:"true"`
+	ImageScanFilter             FieldsFilter `json:"image_scan_filter" required:"true"`
+	ContainerScanFilter         FieldsFilter `json:"container_scan_filter" required:"true"`
+	HostScanFilter              FieldsFilter `json:"host_scan_filter" required:"true"`
+	CloudAccountScanFilter      FieldsFilter `json:"cloud_account_scan_filter" required:"true"`
+	KubernetesClusterScanFilter FieldsFilter `json:"kubernetes_cluster_scan_filter" required:"true"`
 }
```

Comment:
Now I am not so sure it makes sense to mix `Compliance` filters with the rest.
`Compliance` only applies to accounts & k8s configurations, and does not scan images/container/host, am I right?

If previous statement is correct, we should add a `CloudScanFilter` like this:
```
type CloudScanFilter struct {
	CloudAccountScanFilter      FieldsFilter `json:"cloud_account_scan_filter" required:"true"`
	KubernetesClusterScanFilter FieldsFilter `json:"kubernetes_cluster_scan_filter" required:"true"`
}
```
This means we will also need to tweak `ScanTriggerCommon` to be only:
```
type ScanTriggerCommon struct {
	NodeIds []NodeIdentifier `json:"node_ids" required:"true"`
}
type ScanTriggerConfigCommon struct {
   ScanTriggerCommon
	CloudScanFilter
}
type ScanTriggerResourceCommon struct {
   ScanTriggerCommon
	ScanFilter
}
```
And replace `ScanTriggerCommon` in the requests.

This will need refactoring in a couple of places, we can do that later if too much work

Reply:
Compliance does scan hosts. We can look into this detail later once Saurabh implements Host/Kubernetes Compliance scans